understanding/20/keyboard-no-exception.html

```diff
@@ -11,11 +11,12 @@ <h1>Understanding Keyboard (No Exception)</h1>
    <section id="brief">
       <h2>In brief</h2>
       <dl>
-         <dt>Objective</dt><dd>Everything can be done with a keyboard</dd>
-         <dt>Author task</dt><dd>Ensure all pointer actions have a keyboard equivalent</dd>
+         <dt>Goal</dt><dd>Everything can be done with a keyboard.</dd>
+         <dt>What to do</dt><dd>Ensure all pointer actions have a keyboard equivalent.</dd>
+         <dt>Why it's important</dt><dd>People who can only use the keyboard need to be able to accomplish everything.</dd> 
```

Comment:
@rainbreaw commented:
> In understanding/20/bypass-blocks.html, understanding/20/focus-order.html, understanding/20/focus-visible.html and understanding/20/keyboard-no-exception.html:

> Consider extending why it's important to include individuals who use switch devices or other AT inputs.

Reply:
Please thumbs up my suggestion in response to Gregg, if you think it solves your concern.